content/en/docs/reference/glossary/managed-control-plane.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Managed Control Plane
+---
+
+Many cloud providers offer a [control plane](/docs/reference/glossary/#control-plane),
```

Comment:
Similarly here, please follow the logical pattern of definitions.
A managed control plane is a control plane that cloud ...